beets/util/__init__.py

```diff
@@ -376,10 +391,10 @@ def bytestring_path(path):
         return path.encode('utf-8')
 
 
-PATH_SEP = bytestring_path(os.sep)
+PATH_SEP: bytes = bytestring_path(os.sep)
 
 
-def displayable_path(path, separator='; '):
+def displayable_path(path, separator: str = '; ') -> str:
```

Comment:
Should be `path: bytes`,  I think. If it's not, that would be a bug in the caller.